stubs/RdKafka/Conf.php

```diff
@@ -86,4 +86,13 @@ public function setConsumeCb(callable $callback)
     public function setOffsetCommitCb(callable $callback)
     {
     }
+
+    /**
+     * @param callable $callback
+     *
+     * @return void
+     */
```

Comment:
Remove this comment. See explanation above.

Reply:
seems to be convention in this file, so i didn't want to break it :D

Reply:
If stubs are staying on php 5.x then definitely it should stay :)